src/language-js/print/literal.js

```diff
@@ -40,14 +43,24 @@ function printLiteral(path, options /*, print*/) {
       }
 
       if (typeof value === "string") {
-        return printString(node.raw, options);
+        return printIndentableString(printString(node.raw, options));
       }
 
       return String(value);
     }
   }
 }
 
+function printIndentableString(str) {
```

Comment:
Use `replaceTextEndOfLine` from '/src/document/utils.js'

Reply:
Nice function, I use it but left ```printIndentableString``` in order to avoid duplicated multiline check code, what do you think?

Reply:
Sorry for not been clear, you can use it directly, no need check `\n`.

Reply:
nice, adjust it